content/doc/book/updating/linux.adoc

```diff
@@ -0,0 +1,102 @@
+---
+layout: section
+title: Linux
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc: left
+:toclevels: 3
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+link:https://www.jenkins.io/doc/book/installing/linux/[Linux installations]
+should be upgraded using the same method as was used for the original installation
+This section will guide you through updating your Jenkins instance to the most recent version.
+
+include::doc/book/updating/_pre-upgrade-wizard.adoc[]
+
+== Upgrade steps on Linux
+[unordered.stack]
+1) Stop your Jenkins Instance :: 
+[source,bash]
+```
+#Stop running Jenkins service using the following command:
+
+sudo su -
+service jenkins stop
```

Comment:
Jenkins installed with Linux installers uses `systemd`.  The `service` command Red Hat Enterprise Linux (and its derivatives) says that it runs System V init scripts.  It does not mention running `systemd` commands.  The Ubuntu 22.04 `service` man page says that it runs both System V init and `systemd` commands.  I think we should use `systemd` rather than using System V init.
```suggestion
systemctl stop jenkins
```